sensor_msgs/Image imgL
sensor_msgs/Image imgR
---
sensor_msgs/Image disparity
